qutebrowser/browser/webengine/webenginetab.py

```diff
@@ -170,6 +248,11 @@ def __init__(self, tab, parent=None):
         super().__init__(tab, parent)
         self._flags = QWebEnginePage.FindFlags(0)  # type: ignore
         self._pending_searches = 0
+        # The API necessary to stop wrapping was added in this version
+        self._wrap_handler = _WebEngineSearchWrapHandler()
+
+    def connect_signals(self):
+        self._wrap_handler.connect_signal(self._widget.page().findTextFinished)
```

Comment:
That's not going to work because the `findTextFinished` attribute gets accessed before the version check here. I'd probably just do something like:

```python3
if qtutils.version_check('5.14'):
    self._widget.page().findTextFinished.connect(self._wrap_handler.store_match_data
```

here (with `store_match_data` being public then), and in `_WebEngineSearchWrapHandler.__init__` do `self._nowrap_available = qtutils_version_check('5.14')` again.

Reply:
:facepalm: When you push the same lines around for a while, they begin to lose meaning - give me one sec